rmw_connextdds_common/include/rmw_connextdds/rmw_waitset_std.hpp

```diff
@@ -333,8 +335,38 @@ class RMW_Connext_StatusCondition : public RMW_Connext_Condition
   virtual bool
   has_status(const rmw_event_type_t event_type) = 0;
 
+  void
+  on_inconsistent_topic(const struct DDS_InconsistentTopicStatus * status);
+
+  void update_status_inconsistent_topic(const struct DDS_InconsistentTopicStatus * status);
+
+  inline rmw_ret_t
+  get_inconsistent_topic_status(
+    rmw_inconsistent_topic_status_t * const status)
+  {
+    update_state(
+      [this, status]() {
+        this->triggered_inconsistent_topic = false;
+
+        status->total_count = this->status_inconsistent_topic.total_count;
+        status->total_count_change = this->status_inconsistent_topic.total_count_change;
+
+        this->status_inconsistent_topic.total_count_change = 0;
```

Comment:
Do we need to clear this variable with `0`? if the next time `total_count_change` is `0`, it should be also updated by `rticonnextdds`?

Reply:
Honestly, I'm not sure.  I pretty much copied this from https://github.com/ros2/rmw_connextdds/blob/806fdfb6e22b587c850bdf153c12a448c6b19e3d/rmw_connextdds_common/include/rmw_connextdds/rmw_waitset_std.hpp#L454